tests/fake_messages/message.json

```diff
@@ -1,39 +1,53 @@
 {
-    "body": {
-        "msg": {
-            "CI_infra_failure" : "DNS",
-            "CI_infra_failure_desc" : "Can't resolve name to IP address",
-            "CI_tier" : "1",
-            "base_distro" : "RHEL 7.2+updates",
-            "brew_task_id" : "11610935",
-            "build_type" : "official",
-            "completion_time" : "2016-08-18T20:55:10Z",
-            "component" : "package-4.4.4-7.el7",
-            "compose_id" : "NULL",
-            "create_time" : "2016-08-18T20:13:51Z",
-            "jenkins_build_url" : "http://domain.local/job/package-4.4.4-7.el7/136/",
-            "jenkins_job_url" : " http://domain.local/job/package-4.4.4-7.el7/",
-            "job_name" : "package-rhel-7.2-candidate-runtest-fslock-func",
-            "owner" : "some_owner",
-            "recipients": ["tbrady", "jedelman"],
-            "team": "someteam",
-            "tests" : [
-                { "executor" : "beaker" , "arch" : "aarch64" , "executed" : "25", "failed" : "0" },
-                { "executor" : "CI-OSP" , "arch" : "x86_64" , "executed" : "1000",   "failed" : "0" }
-            ],
-            "trigger" : "git push"
-        },
-        "topic": "/topic/CI"
-    },
+  "body": {
     "headers": {
```

Comment:
OK.  I *think* there are not headers *inside* the body.  That is probably an artifact introduced by datagrepper.  I think when the messages show up to `consume` they only appear *alongside* the body.

Reply:
FYI - @mprahl presented evidence to the contrary in IRC.